d2graph/d2graph.go

```diff
@@ -151,25 +169,42 @@ func (r Reference) InEdge() bool {
 }
 
 type Style struct {
-	Opacity      *Scalar `json:"opacity,omitempty"`
-	Stroke       *Scalar `json:"stroke,omitempty"`
-	Fill         *Scalar `json:"fill,omitempty"`
-	FillPattern  *Scalar `json:"fillPattern,omitempty"`
-	StrokeWidth  *Scalar `json:"strokeWidth,omitempty"`
-	StrokeDash   *Scalar `json:"strokeDash,omitempty"`
-	BorderRadius *Scalar `json:"borderRadius,omitempty"`
-	Shadow       *Scalar `json:"shadow,omitempty"`
-	ThreeDee     *Scalar `json:"3d,omitempty"`
-	Multiple     *Scalar `json:"multiple,omitempty"`
-	Font         *Scalar `json:"font,omitempty"`
-	FontSize     *Scalar `json:"fontSize,omitempty"`
-	FontColor    *Scalar `json:"fontColor,omitempty"`
-	Animated     *Scalar `json:"animated,omitempty"`
-	Bold         *Scalar `json:"bold,omitempty"`
-	Italic       *Scalar `json:"italic,omitempty"`
-	Underline    *Scalar `json:"underline,omitempty"`
-	Filled       *Scalar `json:"filled,omitempty"`
-	DoubleBorder *Scalar `json:"doubleBorder,omitempty"`
+	Opacity       *Scalar `json:"opacity,omitempty"`
+	Stroke        *Scalar `json:"stroke,omitempty"`
+	Fill          *Scalar `json:"fill,omitempty"`
+	FillPattern   *Scalar `json:"fillPattern,omitempty"`
+	StrokeWidth   *Scalar `json:"strokeWidth,omitempty"`
+	StrokeDash    *Scalar `json:"strokeDash,omitempty"`
+	BorderRadius  *Scalar `json:"borderRadius,omitempty"`
+	Shadow        *Scalar `json:"shadow,omitempty"`
+	ThreeDee      *Scalar `json:"3d,omitempty"`
+	Multiple      *Scalar `json:"multiple,omitempty"`
+	Font          *Scalar `json:"font,omitempty"`
+	FontSize      *Scalar `json:"fontSize,omitempty"`
+	FontColor     *Scalar `json:"fontColor,omitempty"`
+	Animated      *Scalar `json:"animated,omitempty"`
+	Bold          *Scalar `json:"bold,omitempty"`
+	Italic        *Scalar `json:"italic,omitempty"`
+	Underline     *Scalar `json:"underline,omitempty"`
+	Filled        *Scalar `json:"filled,omitempty"`
+	DoubleBorder  *Scalar `json:"doubleBorder,omitempty"`
+	TextTransform *Scalar `json:"capsLock,omitempty"`
+}
+
+func (s Style) NoTextTransform() bool {
```

Comment:
subtle, but this should be `NoneTextTransform`. it's a special value set, otherwise can be confused with having no value (e.g. s.TextTransform == nil)

Reply:
or i guess this is moot given the comment about inlining these instead

Reply:
@alixander -  I'm happy to inline _all_ of them **except this one** - as this is being called from a few places. I can rename this to `NoneTextTransform` to avoid having this `if` clause all over the code. 

Will make life much easier once we come back to refactoring these to not all use `d2graph.Scalar`.

You OK with this?

Reply:
oh i see. yeah that's fine